### ISSUE TITLE
Should autohint be set to off by default?

### ISSUE THREAD

**Issue by guiniol** [bohoomil/fontconfig-ultimate#177]
Hello all!

I'm certainly no font wizard, and I usually install fontconfig-ultimate, leave the defaults and roll with it.

However, for the last few days/weeks, the fonts used by pdf.js in firefox to display pdfs was awful: http://imgur.com/a/JFP8M.
Today, I finally got around to fixing it. After much, tinkering, what fixed it was setting `autohint` to `false` in `/etc/fonts/conf.avail.infinality/10-base-rendering.conf`. From what I read afterwards, it seems that `autohint` is only useful for older fonts and can cause newer fonts to be less beautiful. If that is the case, shouldn't `autohint` be set to false by default? What will I lose by setting it to false? For info, it is using the 'Noto Sans' font.

Keep in mind that something else started this because the last fontconfig update was in may (using archlinux here), and that it may be just me not understanding what is going on. Also, not a lot of people are affected because I couldn't find anyone else complaining.

If I understood anything wrong, or if there is another solution, I'm all ears.

Cheers,

**Comment by zhou13** [comment]
Agree.  This breaks all users' firefox and hard to find the actual reason!

**Comment by Kubuxu** [comment]
I was complaining long time ago, I just didn't found the original cause: https://github.com/mozilla/pdf.js/issues/7494

**Comment by guiniol** [comment]
Reading that thread, the issue may have started when I removed the xf86 intel drivers in favor of the generic ones. There had been some problematic pdfs before that but most were fine, while lately every pdf was problematic, even those that displayed fine before.
